fluss-flink/fluss-flink-common/src/main/java/org/apache/fluss/flink/source/enumerator/FlinkSourceEnumerator.java

```diff
@@ -580,9 +581,8 @@ private List<SourceSplitBase> generateHybridLakeFlussSplits() {
             pendingHybridLakeFlussSplits = lakeSplitGenerator.generateHybridLakeFlussSplits();
             return pendingHybridLakeFlussSplits;
         } catch (Exception e) {
-            LOG.error("Failed to get hybrid lake fluss splits, won't take splits in lake.", e);
+            throw new FlinkRuntimeException("Failed to generate hybrid lake fluss splits", e);
```

Comment:
Let's just throw exception to notify users generate lake splits fails which should be exceptions happens that should be resolved by users.